### DISCUSSION TITLE
What is the purpose of self._persist_coordinator_model_strings_in_db()

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1303] [Q&A] [answered]
as we have implemented our own Database, and this self._persist_coordinator_model_strings_in_db() call is done from the initialize() I would like to understand if that is required or not.

**Comment by TheJulianJES** [comment, marked as answer]
puddly can probably provide you with a more detailed answer later, but ZHA requires this to split its initialization into two phases. The first phase already needs to the "coordinator info" (hence it's saved to the DB). See the PR descriptions for more info:
- https://github.com/zigpy/zigpy/pull/1238
- https://github.com/home-assistant/core/pull/98082
